aten/src/THC/generic/THCTensorMath.cu

```diff
@@ -271,7 +271,7 @@ void THCTensor_(nonzero)(THCState* state, THCudaLongTensor *tensor,
   self = THCTensor_(newContiguous)(state, self);
   thrust::device_ptr<real> self_data(THCTensor_(data)(state, self));
 
-  int num_dim = THCTensor_(nDimension)(state, self);
+  int num_dim = THCTensor_(_nDimension)(state, self);
```

Comment:
There still are many other unintended changes. Here, onnx submodule, `nccl/CMakeList.txt`, `cudnn_extension.cpp`, etc.

Reply:
@SsnL  These are changes that were auto-merged in, not changes I made personally. Per our conversation, I'm going to hard reset back to the original pull request. I'll put any content changes that you've specified in an additional PR.